luaui/Widgets/unit_auto_group.lua

```diff
@@ -201,12 +213,44 @@ local function RemoveOneUnitFromGroupHandler(_, _, args)
 	return true
 end
 
+local function loadAutogroupPreset(_, _, args, data)
+	local pr = args[1]
```

Comment:
"pr" is a bit cryptic
```suggestion
	local newPreset = tonumber(args[1])
```
(and associated slight refactor)